docs/manual-EN/3.build-develop-and-administration/2.install/deploy-nebula-with-swarm.md

```diff
@@ -0,0 +1,217 @@
+# Deploy Nebula Graph cluster with Docker Swarm
+
+This document gives the introduction on deploying a Nebula Graph cluster with Docker Swarm.
+
+## Prerequisites
+
+Before deploying cluster, make sure that you have installed Nebula Graph, Docker, Docker Compose. To customize load balancing and high availability, you need to install HAProxy, and Keepalived.
+
+The hosts used in this document are as follow:
+
+| IP            | Memory (GB) | CPU (CORES) | Role    |
+| ------------- | ----------- | ----------- | ------- |
+| 192.168.1.166 | 16          | 4           | manager |
+| 192.168.1.167 | 16          | 4           | worker  |
+| 192.168.1.168 | 16          | 4           | worker  |
+
+## Create Nebula Graph cluster
+
+### Initialize the Docker Swarm cluster
```

Comment:
Please consider transforming H3 titles in the Create Nebula Graph cluster section into an ordered list. That should provide a clearer view and the readers will know the execution orders at a glance.
The steps below the H3 titles can be substeps.